web_company_color/models/res_company.py

```diff
@@ -19,6 +19,7 @@ class ResCompany(models.Model):
           background: %(color_navbar_bg)s !important;
           background-color: %(color_navbar_bg)s !important;
           color: %(color_navbar_text)s !important;
+          border-bottom: 0px solid %(color_navbar_bg)s !important;
```

Comment:
What is this doing? Adding `!important` should be the latest option always. Before, use a specific selector for putting where you want it.